specs/protocol/proposals.md

```diff
@@ -1,181 +1,14 @@
-for trustless execution of L2-to-L1 messaging, such as withdrawals.
-These output proposals act as the bridge's view into the L2 state.
-Actors called "Proposers" submit the output roots to the settlement layer (L1) and can be contested with a fault proof,
-with a bond at stake if the proof is wrong. The [op-proposer][op-proposer] in one such implementation of a proposer.
-
-[op-proposer]: https://github.com/ethereum-optimism/optimism/tree/d48b45954c381f75a13e61312da68d84e9b41418/op-proposer
-
-_Note_: Fault proofs on Optimism are not fully specified at this time. Although fault proof
-construction and verification [is implemented in Cannon][cannon],
-the fault proof game specification and integration of a output-root challenger into the [rollup-node][g-rollup-node]
-are part of later specification milestones.
-
-[cannon]: https://github.com/ethereum-optimism/cannon
-
-## Proposing L2 Output Commitments
-
-The proposer's role is to construct and submit output roots, which are commitments to the L2's state,
-to the `L2OutputOracle` contract on L1 (the settlement layer). To do this, the proposer periodically
-queries the [rollup node](rollup-node.md) for the latest output root derived from the latest
-[finalized](rollup-node.md#finalization-guarantees) L1 block. It then takes the output root and
-submits it to the `L2OutputOracle` contract on the settlement layer (L1).
-
-### L2OutputOracle v1.0.0
-
-The submission of output proposals is permissioned to a single account. It is expected that this
-account will continue to submit output proposals over time to ensure that user withdrawals do not halt.
-
-The [L2 output proposer][op-proposer] is expected to submit output roots on a deterministic
-interval based on the configured `SUBMISSION_INTERVAL` in the `L2OutputOracle`. The larger
-the `SUBMISSION_INTERVAL`, the less often L1 transactions need to be sent to the `L2OutputOracle`
-contract, but L2 users will need to wait a bit longer for an output root to be included in L1 (the settlement layer)
-that includes their intention to withdraw from the system.
-
-The honest `op-proposer` algorithm assumes a connection to the `L2OutputOracle` contract to know
-the L2 block number that corresponds to the next output proposal that must be submitted. It also
-assumes a connection to an `op-node` to be able to query the `optimism_syncStatus` RPC endpoint.
-
-```python
-import time
-
-while True:
-    next_checkpoint_block = L2OutputOracle.nextBlockNumber()
-    rollup_status = op_node_client.sync_status()
-    if rollup_status.finalized_l2.number >= next_checkpoint_block:
-        output = op_node_client.output_at_block(next_checkpoint_block)
-        tx = send_transaction(output)
-    time.sleep(poll_interval)
-```
-
-A `CHALLENGER` account can delete multiple output roots by calling the `deleteL2Outputs()` function
-and specifying the index of the first output to delete, this will also delete all subsequent outputs.
-
-## L2 Output Commitment Construction
```

Comment:
I think we can leave this section in here, the definition of the output since it didn't change with fault proofs